api/client/src/test/java/org/projectnessie/client/auth/oauth2/TestOAuth2ClientConfig.java

```diff
@@ -312,12 +313,12 @@ static Stream<Arguments> testFromConfig() {
             ImmutableMap.of(
                 CONF_NESSIE_OAUTH2_TOKEN_ENDPOINT, "https://example.com/token",
                 CONF_NESSIE_OAUTH2_CLIENT_SECRET, "s3cr3t",
-                CONF_NESSIE_OAUTH2_REFRESH_SAFETY_WINDOW, "PT10S",
```

Comment:
The previous test for empty client ID was actually a test for the `@Check` method and is already covered above (cf. line125), so I removed it.